doc/Development_Documentation/05_Objects/01_Object_Classes/01_Data_Types/10_Calculated_Value_Type.md

```diff
@@ -64,7 +64,7 @@ You can also provide a Symfony service as calculator class via `@` prefix (e.g.
 
 Getter methods on the object class are generated as usual. The code to retrieve the values would then be: 
 ```php
-$object = Object_Abstract::getByPath("/demoobject");
+$object = DataObject::getByPath("/demoobject");
```

Comment:
Should be use `AbstractObject` here, as `DataObject` is just an alias, wdyt?

Reply:
Sure, if that's preferred.